src/carousel/carousel.scss

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+@import '../button/button';
+
+@mixin iui-carousel {
+  display: flex;
+  flex-direction: column-reverse;
+  overflow: hidden;
+}
+
+@mixin iui-carousel-slider {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: transform $iui-speed-slow ease-in-out;
+  }
+
+  &-item {
+    position: relative;
+    width: 100%;
+    flex-shrink: 0;
+    box-sizing: border-box;
+  }
+}
+
+@mixin iui-carousel-navigation {
+  display: flex;
+  align-items: center;
+  height: $iui-baseline * 3;
+  @include themed {
+    border-top: $iui-xxs solid t(iui-color-background-4);
+  }
+
+  &-dots {
+    @include iui-reset;
+    display: flex;
+    flex-grow: 1;
+    white-space: nowrap;
+    overflow: hidden;
+    justify-content: center;
+    list-style: none;
+  }
+
+  &-dot {
+    @include iui-button;
+    @include iui-button-size($size: small);
+    background-color: transparent;
+
+    &:hover:not(.iui-active)::after {
+      @include themed {
+        background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-3));
+      }
+    }
+
+    &.iui-invisible {
+      cursor: default;
+      display: none;
+    }
+
+    &:focus {
+      outline: none;
+
+      &::after {
+        @include themed {
+          outline: 1px solid t(iui-color-foreground-primary);
+          outline-offset: 1px;
+        }
+      }
+    }
+
+    &:focus:not(:focus-visible)::after {
+      @include themed {
+        outline: none;
+      }
+    }
+
+    &::after {
+      content: '';
+      display: block;
+      width: $iui-icons-small;
+      height: $iui-icons-small;
+      border-radius: 50%;
+      transition: background-color $iui-speed-fast ease;
+      @media (prefers-reduced-motion: no-preference) {
+        transition: background-color $iui-speed-fast ease, transform $iui-speed-fast ease;
+      }
+      @include themed {
+        background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
+      }
+    }
+
+    &.iui-first::after {
+      transform: scale(0.5);
+    }
+
+    &.iui-second::after {
```

Comment:
Do you think we could rename first and second into something else? Cause these dots are at the end, they are inverted order :D all they differ is in size, so maybe could figure some fancy wording? 
If not, its ok, minor

Reply:
I proposed `.iui-small`, `.iui-smaller`, & `.iui-smallest` instead of `.iui-second`, `.iui-first`, & `.iui-invisible` but it wasn't received well.

Another idea is applying `.iui-truncate-start` and / or `.iui-truncate-end` on the `<ol>` to target the nth child, but the hidden `<li>` are still in the DOM so that would require some re-working.

Reply:
amm. I would stick with having classes better.. looks much simpler...But cannot propose anything better than `small` namings...
Another question then, do all dots need to be in the dom, if they are invisible? Shouldnt be enough to render only those visible? Is there something about that in w3? Cause then maybe we could simplify css selectors even without classes...

Reply:
I think the accessibility benefit here would be that the screen reader would announce how many dots are in the list.

It might also help to keep them in the DOM for animation purposes (if we want to add animation)

Reply:
Yeah, and then someone comes in with a lot of images and we render lots of empty items :)
I am not that sure about it.. Maybe there are some attributes to say the amount instead..

Reply:
we would still have lots of slides in the DOM, which I think is a bigger problem 😆

Reply:
I assumed we gonna have just 3 slides and switch content inside :D

Reply:
> I assumed we gonna have just 3 slides and switch content inside :D

Wouldn't that be an issue if you are at the last slide & click the next button which sends you back to the very first slide (animating back past all of the previous slides)?

Reply:
oh, I thought you cannot go beyond last slide :O Is this recommended UX?

Reply:
We don't have clear guidance on if looping back from the last slide to the first slide is good UX or not.  w3 does do this though.

I could see home page carousels / image carousels finding this useful, but I could also find this being bad for a welcome / onboarding tutorial.  Maybe we give this as an option within React?